pages/about.js

```diff
@@ -10,23 +10,20 @@ import {
 import Layout from '../components/Layout'
 import NavBar from '../components/NavBar'
 
-export default class About extends React.Component {
-  render () {
+const About = () => {
+  return (
+    <Layout>
+      <Head>
+        <title>About OONI Explorer</title>
+      </Head>
 
-    return (
-      <Layout>
-        <Head>
-          <title>About OONI Explorer</title>
-        </Head>
-
-        <NavBar />
-
-        <Container>
-          <Heading h={2}>XXX Implement Me</Heading>
-          <Text>Do we even need an about page?</Text>
-        </Container>
-      </Layout>
-    )
-  }
+      <NavBar />
 
+      <Container>
+        <Heading h={2}>XXX Implement Me</Heading>
+        <Text>Do we even need an about page?</Text>
+      </Container>
+    </Layout>
+  )
 }
+export default About
```

Comment:
Should we maybe at this point just drop this page or setup a redirect to ooni.org/about?

Reply:
I have removed the about page, we didn't link to it anywhere and there is already a link to ooni.org/about in the footer.